### PR TITLE
Add text-ellipsis to navlink

### PR DESCRIPTION
**Pull request by Yxwww**
resolves: https://github.com/curvenote/curvespace/issues/25
- add title to show the full title

image

### CODE REVIEW

File: app/components/Navigation/LeftNav.tsx [hunk 1]
```diff
@@ -27,8 +29,9 @@ const HeadingLink = ({
   return (
     <NavLink
       prefetch="intent"
+      title={title}
       className={({ isActive }) =>
-        classNames('block', {
+        classNames('block text-ellipsis overflow-hidden', {
```
Comment (stevejpurves) [inline app/components/Navigation/LeftNav.tsx:34]: if we are using ellipsis then we should have a tooltip with the full content available.

Comment (rowanc1) [reply app/components/Navigation/LeftNav.tsx:34]: Updated the description of the original issue @stevejpurves, I think the whole title is important to show, and that the example you showed had a long title, so adding something like `break-word` for overflow wrapping will fix it?

 `overflow-wrap: break-word;`

Comment (stevejpurves) [reply app/components/Navigation/LeftNav.tsx:34]: 👍🏼  on `break-word` here

**Comment by Yxwww** [comment]
Break word does the job:

Screen Shot 2022-04-22 at 3 26 15 PM

**Comment by rowanc1** [comment]
Looks good. Also putting this on the title while we are at it:

image

**Pull request merged**